src/pages/filter/refiner/refiner.page.tsx

```diff
@@ -29,10 +33,12 @@ export const RefinerPage = observer((): ReactElement => {
   useDatasetName()
 
   useEffect(() => {
+    presetName && applyPreset(presetName)
```

Comment:
For variant drawer i moved similar logic to the store. see `variantStore.observeVarianthistory`
It doesn't matter, but may be, we can discuss general style for history control